docker-compose.dev.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:12.3
+    container_name: google-scraper_db
+    environment:
+      - POSTGRES_DB=google_scraper_development
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./docker_postgres_init.sql:/docker-entrypoint-initdb.d/docker_postgres_init.sql
```

Comment:
Do we actually need this? Could you please explain more why you decide to keep this file?

Reply:
@gutakk This file would be used to initialize the default database so later when we have models we can add the database table to this file, clean the docker container and rerun the composer to build the database with the new table.
More info about the initialization script can be found on the `Initialization scripts` section on [this document](https://hub.docker.com/_/postgres/) about Postgres image 🙌🏼

Reply:
This sounds like migration, isn't it? If yes, does Beego has their own migration?

Reply:
@gutakk Oh I see what you mean now I didn't know that Beebo has their own migration 😢